addons/purchase_stock/wizard/purchase_order_suggest.py

```diff
@@ -158,13 +158,18 @@ def action_purchase_order_suggest(self):
             'infos': {'refresh': True},
         }
 
-    def _get_montlhy_demand_context(self):
+    def _get_suggested_products_context(self):
         self.ensure_one()
-        start_date, limit_date = self._get_period_of_time()
-        context = {
-            'monthly_demand_start_date': start_date,
-            'monthly_demand_limit_date': limit_date,
-        }
+        if self.based_on == 'actual_demand':
+            context = {
+                'number_of_days_for_replenish': self.number_of_days
```

Comment:
If we use `outgoing_qty` instead of `actual_demand`:
```suggestion
                'from_date': fields.Datetime.now(),
                'to_date': fields.Datetime.now() + relativedelta(days=self.number_of_days),
```